HaikuArchives/DebugMonitor
# Fix build: x86_64

Should build on x86_64. Don't know about x86 or x86_gcc2.
This is for an upcoming recipe :)

How I built/tested the source:
* There are 3 directories. Run `make` in each of them.
* Go to DebugMonitor and run the build. No windows should appear.
* Go to Scribble and run the build. Click on the main window. A DebugMonitor window should appear.
* Quit Scribble. There should be a new entry on the DebugMonitor window.
